kore/src/Kore/Simplify/Simplify.hs

```diff
@@ -142,10 +142,13 @@ data Env = Env
         SideCondition RewritingVariableName ->
         TermLike RewritingVariableName ->
         Simplifier (OrPattern RewritingVariableName)
-    , simplifierAxioms :: !BuiltinAndAxiomSimplifierMap
+    , -- | A map with the user defined evaluators.
+      -- Builtin evaluators are not stored in this map.
+      simplifierAxioms :: !BuiltinAndAxiomSimplifierMap
```

Comment:
(nit) Just stumbled over this type name here - no built-in things in the `BuiltInAndSomething` any more... 
Maybe worth renaming if anything else needs to still change (or when updating the branch to master state).

Reply:
Good catch, thanks! Since I'm now trying to avoid storing user-defined evaluators in maps too, I'll just delete this type once I'm done.